.github/workflows/ci.yml

```diff
@@ -189,7 +189,9 @@ jobs:
       - name: Install python wheel package
         run: pip install wheel
       - name: Install python requirements
-        run: pip install -r build-requirements.txt .[data]
+        run: |
+          pip install -r build-requirements.txt .[data]
+          pip install -r build-requirements.txt .[docker]
```

Comment:
I think you can just do `pip install -r build-requirements.txt .[data,docker]` here.